app_dart/lib/src/service/config.dart

```diff
@@ -154,6 +154,9 @@ class Config {
   /// Upper limit of targets to be backfilled in API call.
   int get backfillerTargetLimit => 50;
```

Comment:
What is the difference between this constant and the new one that was added?

Reply:
`backfillerTargetLimit` is the number limit of targets to be call in each API call. Most of the cases, we have hundreds of targets to schedule, but enforce this limit to avoid potential backend errors.

The new `backfillerCommitLimit` is the number of commit rows to scan (each row contains the to-be-scheduled targets). From task execution point of view we hope to scan as many as possible when bots are idle. But it would be a waste of resource to check say 1000 rows each time. 50 should be a big enough number to start with.

Updated the doc to explain.

Reply:
Ah okay. Thanks for adding the doc.